integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/kotlin/StdlibGradleIntegrationTest.kt

```diff
@@ -17,7 +17,7 @@ class StdlibGradleIntegrationTest(override val versions: BuildVersions) : Abstra
         @get:JvmStatic
         @get:Parameterized.Parameters(name = "{0}")
         val versions = BuildVersions.permutations(
-            gradleVersions = listOf("5.6"),
+            gradleVersions = listOf("6.7.1"),
```

Comment:
This is the main change and reason for this PR. Looks like Gradle 5 does not support Kotlin DSL plugin, see #2702 

I think our minimum supported Gradle version should be the same as in KGP, which is 6.7.1, so there's no point testing against 5.6